common/experiment_path.py

```diff
@@ -23,11 +23,11 @@ def path(*path_segments) -> Path:
     return Path(experiment_utils.get_work_dir(), *path_segments)
 
 
-def gcs(path_obj: Path) -> str:
-    """Returns a string with WORK_DIR replaced with |bucket|. |path_obj| should
-    be created by path()."""
+def filestore(path_obj: Path) -> str:
```

Comment:
These changes look like previous commits on master. Consider rebasing the `klee` branch onto `master` first to get rid of these unrelated changes.